### PR TITLE
Sign In and Sign Out

### PR DESCRIPTION
**Pull request by BoyKommissarr**
Links added for Signing In, Signing out, Logging In and Logging Out.

### CODE REVIEW

**Comment by BoyKommissarr** [comment]
Linked Sign In, Sign Out, Log In and Log Out.

File: app/views/shared/_footer.html.erb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <footer class="mt-auto text-white-50">
-    <p>Cover template for <a href="https://getbootstrap.com/" class="text-white">Bootstrap</a>, by <a href="https://twitter.com/mdo" class="text-white">@mdo</a>.</p>
-  </footer>
\ No newline at end of file
+  <p>Tooba Masjid &copy 2024</p>
```
Comment (junaidkhan2024) [inline app/views/shared/_footer.html.erb:2]:
```suggestion
  <p>Tooba Masjid &copy <%= DateTime.now.year %></p>
```

Can we make it this dynamic ?

File: app/views/welcome/index.html.erb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <main class="px-3" >
     <center><img height="100px" width="40%" src="https://as2.ftcdn.net/v2/jpg/08/57/77/49/1000_F_857774949_IIWZ0v2RcYdALb9HM1y5NURNuj33sEEH.jpg"></center><br>
 
-    <h1> Assalam Alaikum .</h1>
-    <p class="lead"> welcome to the "Way to masjid" App here we can guide you to the nearest moasjid and its salah timing.</p>
+    <h1> Assalam-o-Alaikum .</h1>
+    <p class="lead"> welcome to the "Way to masjid" App here we can guide you to the nearest masjid and its salah timing.</p>
```
Comment (junaidkhan2024) [inline app/views/welcome/index.html.erb:5]:
```suggestion
    <p class="lead"> Welcome to the "Way to masjid" App here we can guide you to the nearest masjid and its salah timing.</p>
```

**Review by junaidkhan2024 — APPROVED** [review]
Code looks good to me but added few suggestions.

File: app/views/shared/_footer.html.erb (hunk 1 above)
Comment (BoyKommissarr) [reply app/views/shared/_footer.html.erb:2]: Changes Done

File: app/views/welcome/index.html.erb (hunk 2 above)
Comment (BoyKommissarr) [reply app/views/welcome/index.html.erb]: Done Changes

**Pull request merged**